{"uid":"80368efcc7cbde80","name":"test_network_cut","fullName":"tests.integration.ha_tests.test_self_healing#test_network_cut","historyId":"6b9f6734f6f55bf616f6dd52eb37e3bd","time":{"start":1718249494802,"stop":1718249796040,"duration":301238},"description":"Completely cut and restore network.","descriptionHtml":"<p>Completely cut and restore network.</p>\n","status":"failed","statusMessage":"AssertionError: Unable to create continuous_writes table","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 390, in test_network_cut\n    await start_continuous_writes(ops_test, app)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/helpers.py\", line 778, in start_continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 347, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 325, in iter\n    raise retry_exc.reraise()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 158, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/helpers.py\", line 786, in start_continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to create continuous_writes table\"\nAssertionError: Unable to create continuous_writes table","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"abort_on_fail","time":{"start":1718249494801,"stop":1718249494801,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1718249494801,"stop":1718249494801,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1718249796041,"stop":1718250099705,"duration":303664},"status":"failed","statusMessage":"AssertionError: Unable to clear up continuous_writes table\nassert 'False' == 'True'\n  \n  - True\n  + False\n","statusTrace":"  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/conftest.py\", line 29, in continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 347, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 325, in iter\n    raise retry_exc.reraise()\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 158, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/postgresql-operator/postgresql-operator/tests/integration/ha_tests/conftest.py\", line 37, in continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to clear up continuous_writes table\"\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1718250099706,"stop":1718250099706,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"fv-az1383-306"},{"name":"thread","value":"8815-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"bbe23b64bef4518c","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":11,"broken":22,"skipped":0,"passed":78,"unknown":0,"total":111},"items":[{"uid":"55f6a4deefe62c24","reportUrl":"../2724//#testresult/55f6a4deefe62c24","status":"passed","time":{"start":1718158709331,"stop":1718159085365,"duration":376034}},{"uid":"2d075d0bbebeb794","reportUrl":"../2717//#testresult/2d075d0bbebeb794","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f0ece969870 state=finished raised AssertionError>]","time":{"start":1717814227626,"stop":1717814408565,"duration":180939}},{"uid":"601cad85d1766cc6","reportUrl":"../2712//#testresult/601cad85d1766cc6","status":"passed","time":{"start":1717727300028,"stop":1717727693764,"duration":393736}},{"uid":"29c28d42e2643e59","reportUrl":"../2707//#testresult/29c28d42e2643e59","status":"passed","time":{"start":1717640312477,"stop":1717640959369,"duration":646892}},{"uid":"3723b2cc3fdb93fd","reportUrl":"../2697//#testresult/3723b2cc3fdb93fd","status":"passed","time":{"start":1717553713817,"stop":1717554412116,"duration":698299}},{"uid":"f36d20fe95247432","reportUrl":"../2687//#testresult/f36d20fe95247432","status":"passed","time":{"start":1717468253218,"stop":1717468871398,"duration":618180}},{"uid":"8835546337e68dc4","reportUrl":"../2679//#testresult/8835546337e68dc4","status":"passed","time":{"start":1717294592337,"stop":1717295154548,"duration":562211}},{"uid":"77b3725e82e4fca5","reportUrl":"../2678//#testresult/77b3725e82e4fca5","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f3eba2eb9d0 state=finished raised AssertionError>]","time":{"start":1717210023075,"stop":1717210230921,"duration":207846}},{"uid":"84603a8208a0dc40","reportUrl":"../2673//#testresult/84603a8208a0dc40","status":"passed","time":{"start":1717121593121,"stop":1717122299818,"duration":706697}},{"uid":"d6b56be3a8063a6f","reportUrl":"../2656//#testresult/d6b56be3a8063a6f","status":"passed","time":{"start":1716863270505,"stop":1716863716496,"duration":445991}},{"uid":"e61b4a6ef6460091","reportUrl":"../2646//#testresult/e61b4a6ef6460091","status":"passed","time":{"start":1716775833374,"stop":1716776255250,"duration":421876}},{"uid":"e7a4fb59f4dafc27","reportUrl":"../2645//#testresult/e7a4fb59f4dafc27","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7fbb5731b1c0 state=finished raised AssertionError>]","time":{"start":1716690941824,"stop":1716691123659,"duration":181835}},{"uid":"e8462dddfe0ca4c4","reportUrl":"../2644//#testresult/e8462dddfe0ca4c4","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f2307ff68f0 state=finished raised AssertionError>]","time":{"start":1716605505991,"stop":1716605728849,"duration":222858}},{"uid":"8e7c64cb05a5cc73","reportUrl":"../2641//#testresult/8e7c64cb05a5cc73","status":"passed","time":{"start":1716516333737,"stop":1716517027227,"duration":693490}},{"uid":"8e5b92d545bf329b","reportUrl":"../2639//#testresult/8e5b92d545bf329b","status":"passed","time":{"start":1716430000050,"stop":1716430399123,"duration":399073}},{"uid":"4fc6e090e5e41ffa","reportUrl":"../2637//#testresult/4fc6e090e5e41ffa","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1716345843963,"stop":1716346149407,"duration":305444}},{"uid":"b04ce7759412d43e","reportUrl":"../2636//#testresult/b04ce7759412d43e","status":"passed","time":{"start":1716260236772,"stop":1716260926316,"duration":689544}},{"uid":"157b99ed8cabd4e6","reportUrl":"../2629//#testresult/157b99ed8cabd4e6","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1716172100627,"stop":1716172404814,"duration":304187}},{"uid":"fe1386f187807cc8","reportUrl":"../2628//#testresult/fe1386f187807cc8","status":"passed","time":{"start":1716085260194,"stop":1716085643233,"duration":383039}},{"uid":"8e6ded197e692664","reportUrl":"../2627//#testresult/8e6ded197e692664","status":"passed","time":{"start":1715997886326,"stop":1715998575531,"duration":689205}}]},"tags":["asyncio","group(1)"]},"source":"80368efcc7cbde80.json","parameterValues":[]}